framework_tvb/tvb/interfaces/web/controllers/spatial/local_connectivity_controller.py

```diff
@@ -140,7 +140,7 @@ def set_equation(self, equation):
         current_lconn.equation = eq_class()
 
         eq_params_form = get_form_for_equation(eq_class)(prefix=LocalConnectivityCreatorForm.NAME_EQUATION_PARAMS_DIV)
-        return {'form': eq_params_form, 'equationsPrefixes': self.plotted_equation_prefixes}
+        return {'adapter_form': eq_params_form, 'equationsPrefixes': self.plotted_equation_prefixes}
```

Comment:
Here you should repeat the solution for the generic form, and have an own controller method to fill a jinja template from the form, instead of relying on __str__